filters/include/pcl/filters/voxel_grid_covariance.h

```diff
@@ -361,14 +361,43 @@ namespace pcl
 
       }
 
+      /** \brief Get the voxels surrounding point p designated by #relative_coordinates.
+       * \note Only voxels containing a sufficient number of points are used.
+       * \param[in] relative_coordinates relative coordinates of neighboring voxels
```

Comment:
Can you provide some intuition on how this variable needs to be structured. I see it represent some notion of adjacency, represented by indices, but I'm clueless on how to supply inputs to this.

Reply:
I refined the comment to provide more information on the relative coordinate matrix structure.